src/pages/journal/components/test-outcome/__tests__/test-outcome.spec.ts

```diff
@@ -167,5 +163,77 @@ describe('Test Outcome', () => {
       });
     });
 
+    describe('show rekey button', () => {
+      it('should show rekey button for a booked test if date is in past', () => {
+        component.slotDetail = testSlotDetail;
+
+        const dateTime = new Date();
+        dateTime.setDate(dateTime.getDate() - 1);
```

Comment:
Please use the same logic to manipulate dates as in the production code (our abstraction is used)
It might be better to fix the date instead of ```new Date()``` and cover the edge cases in separate scenarios